waspc/examples/todoApp/src/App.tsx

```diff
@@ -1,10 +1,12 @@
-import { useSocket } from "wasp/client/webSocket";
-import { Link } from "wasp/client/router";
-import { logout, useAuth } from "wasp/client/auth";
-import { useQuery, getDate } from "wasp/client/operations";
+import { useSocket } from 'wasp/client/webSocket'
+import { Link } from 'wasp/client/router'
+import { logout, useAuth } from 'wasp/client/auth'
+import { useQuery, getDate } from 'wasp/client/operations'
 
 import './Main.css'
 import { getName } from './user'
+// Necessary to trigger type tests.
+import './TestRpcTypes'
```

Comment:
I'd recommend maybe creating a `type-tests` folder or something like that and then import a `index.ts` from that folder to keep things a bit more organised.

Reply:
Then you could put the helpers in `typeTestHelpers` or something like that and then the RPC tests just in its own file.